functions/zram.bash

```diff
@@ -41,26 +41,28 @@ init_zram_mounts() {
       fi
     fi
 
-    if ! dpkg -s 'make' &> /dev/null; then
-      echo -n "$(timestamp) [openHABian] Installing zram required package (make)... "
-      if cond_redirect apt-get install --yes -o DPkg::Lock::Timeout="$APTTIMEOUT" make; then echo "OK"; else echo "FAILED"; return 1; fi
+    if ! dpkg -s 'meson' &> /dev/null; then
+      echo -n "$(timestamp) [openHABian] Installing zram required package (meson)... "
+      if cond_redirect apt-get install --yes -o DPkg::Lock::Timeout="$APTTIMEOUT" meson; then echo "OK"; else echo "FAILED"; return 1; fi
     fi
 
     install_zram_code "$zramInstallLocation"
 
     echo -n "$(timestamp) [openHABian] Setting up OverlayFS... "
-    if ! cond_redirect make --always-make --directory="$zramInstallLocation"/zram-config/overlayfs-tools; then echo "FAILED (make overlayfs)"; return 1; fi
-    if ! cond_redirect mkdir -p /usr/local/lib/zram-config/; then echo "FAILED (create directory)"; return 1; fi
-    if cond_redirect install -m 755 "$zramInstallLocation"/zram-config/overlayfs-tools/overlay /usr/local/lib/zram-config/overlay; then echo "OK"; else echo "FAILED (install overlayfs)"; return 1; fi
+    if ! cond_redirect rm -rf "$zramInstallLocation"/zram-config/overlayfs-tools/builddir; then echo "FAILED (remove builddir)"; return 1; fi
```

Comment:
return-if-fail isnt required, is it?
(not sure but I think even with rm -f the cmd will exit with code != 0 if the dir does not exist)

Reply:
True, I could simplify that. Will do